src/DistortableCollection.js

```diff
@@ -100,6 +101,15 @@ L.DistortableCollection = L.FeatureGroup.extend({
       /* conditional prevents disabled images from flickering multi-select mode */
       if (layer.editing.enabled()) {
         L.DomUtil.toggleClass(e.target, 'collected');
+        // re-order layers by ... to match their display order in UI
+        const newArr = arr.every((each) => {
+          return each._leaflet_id !== layer._leaflet_id;
+        });
+        if (newArr) {
```

Comment:
Here can we add a comment like "Add new layers to the end, or insert existing layers at the correct location"?

Reply:
Sure we can @jywarren, will do